### ISSUE TITLE
very interesting 

### ISSUE THREAD

**Issue by mishav78** [samrandhaji/Undetectable-AI#1]
I looked at main. does the system work on Google translate or more? Because I'm not sure Google translate would be ?

**Comment by samrand96** [comment]
Yes definitely it works with Google Translator but I am currently working on another model that will eliminate the Google Translator and also the without ollama and will only needs couple of MB RAM to run with T5 Model

**Comment by mishav78** [comment]
is the new model based on getting synonyms from T5?

**Comment by mishav78** [comment]
please answer
